### PR TITLE
add door status to nav bar

### LINKED ISSUE #14: Door status on website
We have this nice door status: http://eingang.metalab.at/ 
It would be really great if it could be included on the website for visitors to see it directly

### PR DESCRIPTION
**Pull request by ehganzlieb**
Fixes #14

### CODE REVIEW

**Comment by Hetti** [comment]
this will not work because of mixed content policy: https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content

**Comment by anlumo** [comment]
Please use https://eingang.c3w.at/ instead. Don't question the history behind that proxy.

**Comment by anlumo** [comment]
This PR needs to wait until eingang.c3w.at has the right CORS headers.

EDIT: already done!

File: location.html [hunk 1]
```diff
@@ -42,11 +42,36 @@ <h2>Hackspace in the Heart of Vienna</h2>
                     c0,4.146,4.69,6.554,8.059,4.138l31.583-22.647C97.418,73.331,97.418,69.118,94.585,67.086z" />
                 </svg>
             </div>
+            <aside class='door-overlay'>Door is <span id="door">loading</span></aside>
+            <!--determine door status-->
+            <script type="text/javascript">
+                fetch("https://eingang.c3w.at/status.json").then((response) => {
+                    if (response.ok && Math.floor(response.status / 100) === 2) {
```
Comment (luto) [inline location.html]: https://developer.mozilla.org/en-US/docs/Web/API/Response/ok

> The ok read-only property of the [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) interface contains a Boolean stating whether the response was successful (status in the range 200-299) or not.

skip the `Math.floor`-magic.

**Review by luto — APPROVED** [review]

**Pull request merged**